torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3411,6 +3411,33 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_conv_transpose1d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # Ordered as shapes for input, weight, bias
+    # and a dict of values of (stride, padding, output_padding, groups, dilation)
+    cases: Tuple[Tuple[int], Tuple[int], Tuple[int], dict] = (  # type: ignore[assignment]
```

Comment:
Nice straightforward case enumeration; I don't have a great sense for what would be interesting sample inputs here but these seem (at least) to be a good start